phase-4-explainer.md

```diff
@@ -0,0 +1,175 @@
+### Phase 4: Full Introspection of an Accessibility Tree
+
+The **Computed Accessibility Tree** API will allow authors to access
+the full computed accessibility tree -
+all computed properties for the accessibility node associated with each DOM element,
+plus the ability to walk the computed tree structure including virtual nodes.
+
+This will make it possible to:
+  * write any programmatic test which asserts anything
+    about the semantic properties of an element or a page.
+  * build a reliable browser-based assistive technology -
+    for example, a browser extension which uses the accessibility tree
+    to implement a screen reader, screen magnifier, or other assistive functionality;
+    or an in-page tool.
+  * detect whether an accessibility property
+    has been successfully applied
+    (via ARIA or otherwise)
+    to an element -
+    for example, to detect whether a browser has implemented a particular version of ARIA.
+  * do any kind of console-based debugging/checking of accessibility tree issues.
+  * react to accessibility tree state,
+    for example, detecting the exposed role of an element
+    and modifying the accessible help text to suit.
+
+#### Why is accessing the computed properties being addressed last?
```

Comment:
Leave this section in the original explainer; link back to this doc from there for "early exploration and ideas"

Reply:
Done.